internal/k8s/pod_test.go

```diff
@@ -0,0 +1,25 @@
+package k8s
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var podsToImagesOut = `blorg-fe-6b4477ffcd-xf98f	blorg.io/blorgdev/blorg-frontend:tilt-361d98a2d335373f
+cockroachdb-0	cockroachdb/cockroach:v2.0.5
+cockroachdb-1	cockroachdb/cockroach:v2.0.5
+cockroachdb-2	cockroachdb/cockroach:v2.0.5
```

Comment:
can you add a test for a pod with more than one container?

Reply:
yes, will do!